src/twisted/conch/ssh/keys.py

```diff
@@ -589,7 +617,10 @@ def _guessStringType(cls, data):
             return 'public_lsh'
         elif data.startswith(b'('):
             return 'private_lsh'
-        elif data.startswith(b'\x00\x00\x00\x07ssh-') or data.startswith(b'\x00\x00\x00\x13ecdsa-'):
+        elif data.startswith(b'\x00\x00\x00\x0bssh-'):  # ED25519
```

Comment:
It will, and I see what you're saying, but I think it's better how it is.  It's a little faster to process and it fits the convention of the rest of the checks. :)